mica-search/src/main/java/org/obiba/mica/search/queries/AbstractDocumentQuery.java

```diff
@@ -195,12 +203,30 @@ protected DocumentQueryJoinKeys processJoinKeys(SearchResponse response) {
     return Lists.newArrayList();
   }
 
+  void getPublishedDocumentsFromHitsByClassName(SearchHits hits, List published, Class defaultClass) {
```

Comment:
Why not just return a list instead of asking to give a list ?
And it's better to define the type of items in the list ...

Reply:
This function is used in StudyQuery and DatasetQuery